### DISCUSSION TITLE
0.6.7 - Custom column names

### DISCUSSION THREAD

**Discussion by roovo** [roovo/obsidian-card-board#117] [Announcements]
# New Features
- customizable column names; for the auto-generated columns (e.g. Untagged, Today, Completed...)
- completed items shown with ~~strike-through~~ text

<hr /><em>This discussion was created from the release <a href='https://github.com/roovo/obsidian-card-board/releases/tag/0.6.7'>0.6.7</a>.</em>

**Comment by NullAleph** [comment]
Hi Roovo,
This is a great plugin that is very useful, so thank you for your work on it.  It would become so much more powerful if the columns were completely customizable and some of the default columns could be removed.  

An example - I use a Kanban approach with columns or buckets that are:  ToDo, Doing, Done, Stuck.  I'd love to use Cardboard in this sort of a task management idea that I believe many people have adopted for their personal Kanbans.

Thanks!
